packages/smooth_app/lib/helpers/background_task_helper.dart

```diff
@@ -0,0 +1,214 @@
+import 'dart:convert';
+import 'dart:io';
+import 'package:openfoodfacts/openfoodfacts.dart';
+import 'package:openfoodfacts/utils/CountryHelper.dart';
+import 'package:smooth_app/database/dao_product.dart';
+import 'package:smooth_app/database/local_database.dart';
+import 'package:smooth_app/query/product_query.dart';
+import 'package:task_manager/task_manager.dart';
+
+const String IMAGE_UPLOAD_TASK = 'Image_Upload';
```

Comment:
Could you add a description for each please.
Also be consistent, some are in uppercase, other in lowercase